app/forms/employing_school_form.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
-class EmployingSchoolForm < TraineeForm
+class EmployingSchoolForm < SchoolForm
   attr_accessor :employing_school_id
 
-  validates :employing_school_id, presence: true
+  validates :employing_school_id, presence: true, unless: -> { searching_again? }
```

Comment:
![image](https://user-images.githubusercontent.com/470137/116688096-18c68880-a9ae-11eb-9904-243c4760d222.png)

deep link dont work...

ie click on `You must choose a school`